src/App/MaterialObject.h

```diff
@@ -42,7 +42,7 @@ class AppExport MaterialObject : public DocumentObject
     App::PropertyMap Material;
 
 
-    /// returns the type name of the ViewProvider
+    /// returns the type name of the ViewProvider  // make constexpr
     const char* getViewProviderName() const override {
```

Comment:
Makes no sense to me: https://stackoverflow.com/questions/34828161/can-virtual-functions-be-constexpr

Reply:
(Side note that this is no longer true in C++20 so it's possible that next year we can do this, and it's possible that it will speed up compile time)